DuckDuckGoTests/WebCacheManagerTests.swift

```diff
@@ -62,9 +63,9 @@ class WebCacheManagerTests: XCTestCase {
     @available(iOS 17, *)
     @MainActor
     func testWhenCookiesHaveSubDomainsOnSubDomainsAndWidlcardsThenOnlyMatchingCookiesRetained() async throws {
-        let logins = MockPreservedLogins(domains: [
-            "mobile.twitter.com"
-        ])
+        let fireproofing = UserDefaultsFireproofing.shared
+        fireproofing.clearAll()
+        fireproofing.addToAllowed(domain: "mobile.twitter.com")
```

Comment:
im not sure i like it, if we ever decide to run our tests concurrently (to speed them up) using a shared store doesn't sound like a good option

Reply:
The problem is the previous mock was a subclass of the actual thing so these tests are a bit 💩  anyway.  Let me see if I can mock it properly.